tlog-tiles.md

```diff
@@ -0,0 +1,157 @@
+
+This document specifies an efficient HTTP API to fetch the signed checkpoint,
+Merkle Tree hashes, and entries of a transparency log containing arbitrary
+entries.
+
+## Conventions used in this document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in [BCP 14][] [RFC 2119][] [RFC
+8174][] when, and only when, they appear in all capitals, as shown here.
+
+[RFC 8446]: https://www.rfc-editor.org/rfc/rfc8446.html
+[BCP 14]: https://www.rfc-editor.org/info/bcp14
+[RFC 2119]: https://www.rfc-editor.org/rfc/rfc2119.html
+[RFC 8174]: https://www.rfc-editor.org/rfc/rfc8174.html
+
+## Parameters
+
+A tiled transparency log is defined by a URL *prefix*, a [checkpoint][] origin,
+and one or more Ed25519 [signed note][] public keys.
+
+The origin line SHOULD be the schema-less URL prefix of the log with no trailing
+slashes. For example, a log with *prefix* `https://rome.ct.example.com/tevere/`
+will use `rome.ct.example.com/tevere` as the checkpoint origin line.
+
+## APIs
+
+The entries in the tree, as well as ancillary information such as proofs and
+signed tree heads are exposed not through dynamic APIs like those defined in
+[RFC 6962][], but as a finite set of static resources which can be fetched with
+HTTP GET requests, and which can be efficiently cached and compressed.
+
+Note that all Merkle tree cryptographic operations are as specified by RFC 6962,
+so these APIs can be thought of as an alternative encoding format for the same
+data.
+
+### Checkpoints
+
+The Signed Tree Head MUST be served as a [checkpoint][] at
+
+	<prefix>/checkpoint
+
+with `Content-Type: text/plain; charset=utf-8`.
+
+This endpoint is mutable, so its headers SHOULD prevent caching beyond a few
+seconds.
+
+The checkpoint MUST carry at least one Ed25519 signature by the log, and MAY
+carry additional signatures of other types.
+
+### Merkle Tree
+
+Instead of serving arbitrary consistency and inclusion proofs, logs serve the
```

Comment:
I'm not sure what makes consistency and inclusion proofs "arbitrary" - what could be more specific to a given entry than the exact path from it to the root (or rather the neighbors on this path)?  Presumably what is meant here is that these proofs are not reusable in the way that tiles are, so maybe this is a good place to state the value of using tiled logs.

Reply:
I meant the client can request proof for arbitrary entries/sizes. Updated.